pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools>=67.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sinol-make"
+version = "0.0.1"
+authors = [
+  { name="Mateusz Masiarz", email="m.masiarz@fri.edu.pl" },
+  { name="Tomasz Nowak", email="tomek.nowak.gm.16@gmail.com" }
+]
+description = "Tool for creating sio2 tasks"
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+	"argparse",
+	"requests",
+  "PyYAML",
```

Comment:
Different indentations. You probably can configure your editor to be consistent.

Reply:
3119b831ce78324e3399fd78ccc91b64245bb9a7